packages/gatsby/index.d.ts

```diff
@@ -26,6 +26,8 @@ export {
   withAssetPrefix,
 } from "gatsby-link"
 
+export { IPluginInfoOptions } from "./src/bootstrap/load-plugins/types"
+
```

Comment:
This line broke TypeScript types as the src folder is not included in the distribution, so you get...

```
node_modules/gatsby/index.d.ts:29:36 - error TS2307: Cannot find module './src/bootstrap/load-plugins/types' or its corresponding type declarations.

29 export { IPluginInfoOptions } from "./src/bootstrap/load-plugins/types"
```

Reply:
Could you open a new issue for this please? :pray:

Reply:
https://github.com/gatsbyjs/gatsby/issues/27573